docs/dev-proc/screenshots.md

```diff
@@ -0,0 +1,54 @@
+# Taking screenshots in firmware
+
+## Introduction
+
+Dasharo EDKII fork comes with the functionality of taking screenshots. This is
+useful for creating guides, how-tos and tests. This document is meant to show
+the best practices for taking screenshots.
+
+## Resolution
+
+When taking screenshots, remember to set the display mode to 80x25. This makes
+the pictures easier to read when embedded into documentation. Notice how this:
+
+![](./images/edk2_frontpage_small.png)
+
+is easier to read than this:
+
+![](./images/edk2_frontpage_big.png)
+
+To set the mode in UEFI:
+
+- Enter the boot menu
+- Select `UEFI Shell`
+- Type in `mode 80 25` and press Enter to switch mode
+- Type in `exit` to go back to boot menu
```

Comment:
i have tried it and first you have to enter the Setup. THen from Onte Time Boot you execute the Shell, set the mode and type Exit to return not the Setup. Otherwise when setup is booted, the resolution is initialized and display ends up in stretched console mode

Reply:
Interesting, when I tested before it worked fine, but now I'm seeing the same issue... Will change.

Reply:
Fixed in 52ada08ae95c92a2b218786fa4c9405bc43023a4